src/infrastructure/repositories/CountryRepositoryImpl.ts

```diff
@@ -36,4 +38,13 @@ export default class CountryRepositoryImpl implements CountryRepository {
     const response = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/adm0/${countryId}/ipc.geojson`);
     return response.json();
   }
+
+  async getCountryGeoJSON(countryName: string): Promise<FeatureCollection> {
+    const mapboxToken = process.env.NEXT_PUBLIC_MAPBOX_ACCESS_TOKEN;
+    const response = await fetch(
+      `https://api.mapbox.com/geocoding/v5/mapbox.places/${countryName}.json?access_token=${mapboxToken}&types=country`
+    );
+    const data = await response.json();
+    return data.features[0];
+  }
```

Comment:
we also have the coordinates for the borders of every country, so we might not need this. Of course, that's a lot more data than just the bounding box, but maybe turf could create the bounding box from that as well?